app/services/open_bureau_date.rb

```diff
@@ -0,0 +1,34 @@
+# require 'singleton'
+
+class OpenBureauDate
+  include DateAndTime::Calculations
+
+  def next_date
+    next_tuesday = Time.zone.today.next_occurring(:tuesday)
+
+    next_date = if first_or_third_in_month?(next_tuesday)
+                  next_tuesday
+                else
+                  next_tuesday + 7
+                end
+
+    next_date.strftime('%d/%m/%Y')
+  end
+
+  def next_date_with_time
```

Comment:
cette méthode me parait overkill, autant le faire dans la vue direct (avec i18n and stuff)